native-image-workshop/README.md

```diff
@@ -124,9 +134,8 @@ Compare that to the time running the app with the regular `java` command:
 time java -cp ./target/graalvmnidemos-1.0-SNAPSHOT-jar-with-dependencies.jar oracle.App
 ```
 
-What do the various parameters we passed to the `native-image` command do? Full documentation on these can be found [here](https://www.graalvm.org/docs/reference-manual/native-image/#image-generation-options):
+What do the various parameters we passed to the `native-image` command do? Full documentation on these can be found [here](https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/):
```

Comment:
Full documentation > The full documentation...

Reply:
updated